### ISSUE TITLE
Comments in enums are broken

### ISSUE THREAD

**Issue by sean-mcmanus** [jeff-hykin/cpp-textmate-grammar#150]
enum enum1
	{
		definition // comment is not recognized
	};
![image](https://user-images.githubusercontent.com/19859882/57410728-77324500-71a0-11e9-98f6-9cba4b863b01.png)

Regression. Also, repros for "enum class" maybe other stuff.

**Comment by jeff-hykin** [comment]
We'll get this fixed by tonight, thanks for posting it.

**Comment by jeff-hykin** [comment]
@matter123 fixed the issue, I merged it in, but I'm having difficulties publishing the extension. I had to create a new token, but even with the new token I'm getting a 401 error. Currently working on solving that.

**Comment by jeff-hykin** [comment]
Alright, the fix published in  `v1.8.10` 👍

**Comment by jeff-hykin** [comment]
<img width="575" alt="Screen Shot 2019-05-08 at 7 28 37 PM" src="https://user-images.githubusercontent.com/17692058/57416845-811b6e80-71c7-11e9-91be-0b1809f19d08.png">
Not quite fully solved yet

**Comment by matter123** [comment]
![Screenshot from 2019-05-08 17-35-58](https://user-images.githubusercontent.com/714007/57417130-26c6e180-71b8-11e9-9265-c611d3ba5db9.png)
Everything works for me with v1.8.11

**Comment by jeff-hykin** [comment]
oh hmm thats weird, I was merging in the testing PR and found the bug when I was working on getting tests passing, I might have caused a regression with it. I'll check that now.

**Comment by matter123** [comment]
so something is definitely off.
the spec for `// comment`
```yaml
  source: //
- scopesEnd:
    - comment.line.double-slash.cpp
  source: ' comment'
- scopesBegin:
    - meta.enum.definition.cpp
  scopes:
    - variable.other.enummember.cpp
```
and the block comments has no comments scope at all

**Comment by jeff-hykin** [comment]
Took me quiet awhile to figure it out but I finally got it. The `v1.8.11` release didn't generate the json/yaml output, so effectively the last change wasn't in there. I noticed the comma pattern was still messed up so I changed the way the enum worked.

**Comment by jeff-hykin** [comment]
okay fully fixed with `v1.8.13`
